www/install.rst

```diff
@@ -26,10 +25,47 @@ which includes all the key packages:
 * `Pyzo <http://www.pyzo.org/>`_: A free distribution based on Anaconda and the
   IEP interactive development environment.  Supports Linux, Windows and Mac.
 
-Linux packages
---------------
+Installing via pip
+------------------
+
+Mac and Linux users can install pre-built binary packages for the scipy stack
+packages using `pip <https://pip.pypa.io/en/stable>`_.  pip is the
+installation tool recommended by the Python Packaging Authority.
+
+At the moment, *pip does not work well for Windows* because we do not have the
+necessary compilers to build some binary packages on Windows.
+
+To install via pip on Mac or Linux, first upgrade pip to the latest version::
+
+    python -m pip install --upgrade pip
+
+Then install the scipy stack packages with pip.  We recommend a *user*
+install, using the ``--user`` flag to pip.  This installs packages for your
+local user, and does not need extra permissions to write to the system
+directories::
+
+    pip install --user numpy scipy matplotlib ipython jupyter pandas sympy nose
+
+For user installs, make sure your user install executable directory is on your
+PATH.
+
+Linux::
 
-Users on Linux can quickly install the necessary packages from repositories.
+    export PATH="$PATH:/home/your_user/.local/bin"
+
+OSX::
+
+    export PATH="$PATH:/Users/your_user/Library/Python/3.5/bin"
```

Comment:
I think that `python -m pip` will use python2/pip2. The executables would then end up in `…/Python/2.7/bin` ?